examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const ini = require("ini");
+const utils = require('../utils');
+const path = require('path');
+const Store = require('../');
+
+// Will be called in context of store
+function writeFile() {
+  console.log("writeFile");
+  utils.mkdir(path.dirname(this.path));
+  fs.writeFileSync(this.path, ini.stringify(this.data), { mode: 0o0600 });
+}
+
+// Will be called in context of store
+function readParseFile() {
+  let data;
+  
+  console.log("readParseFile");
+  try
+  {
+    data = fs.readFileSync(this.path, "utf-8");
+  }
+  catch (e)
+  {
+    console.log(`readParseFile error; starting with empty data`);
+    data = {};
+  }
+
+  // Parse the INI-format configuration file
+  return ini.parse(data);
+}
+
+const store = new Store(
```

Comment:
Please update this so that `'app'` and the starting `{` are on the same line, then you'll be able to de-indent the rest of the options object.